tasklistapiOAS.yaml

```
@@ -0,0 +1,130 @@
+openapi: 3.0.3
+info:
+  title: Task List App API
+  version: 1.0.0
+  description: API for managing tasks in a task list application
+  contact:
+    name: Jackson Geer
+    email: jacksongeer@gmail.com
+    url: https://github.com/skeptickal/TaskListBackend
+servers:
+  - url: "http://localhost:8080"
+paths:
+  /tasks:
+    get:
+      tags:
+        - Task List App Functions
+      summary: Get all tasks
```

I would change this to something like:

```suggestion
      summary: Get all tasks or all tasks matching a given status
```

---

done